spec/index.html

```diff
@@ -442,39 +444,40 @@ <h3>Quoted Literals</h3>
-        (<span class="codepoint">U+0040</span>).
+        If present, the <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a> is preceded by
+        an '<code>@</code>' (at sign, <code class="codepoint">U+0040</code>).
         If there is no language tag, there may be a <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>,
         preceded by two concatenated <code>^</code> characters,
         each having the code point <span class="codepoint">U+005E</span>.
         The datatype IRI in Turtle may be written using either a
-        <a data-cite="RFC3986#section-5">resolved</a> IRI,
-        a <a data-cite="RDF12-CONCEPTS#dfn-relative-iri">relative IRI reference</a>,
-        or a <a>prefixed name</a>.
+        <a href="#sec-iri">resolved IRI</a>,
+        a <a href="#relative-iri">relative IRI reference</a>,
+        or a <a href="#prefixed-name">prefixed name</a>.
         If there is no datatype IRI and no language tag,
         the datatype is <code>xsd:string</code>.
       </p>
 
-      <p>A backslash ('<code>\</code>', <span class="codepoint">U+005C</span>)
+      <p>A '<code>\</code>' (backslash, code point <span class="codepoint">U+005C</span>)
         may not appear in any quoted literal except as part of an escape sequence.
         Other restrictions depend on the delimiter:</p>
 
       <ul>
-        <li>Literals delimited by <code>'</code> (<span class="codepoint">U+0027</span>)
-          may not contain unescaped <code>'</code>,
-          <code title="LINE FEED"><sub>LF</sub></code> (<span class="codepoint">U+000A</span>),
-          nor <code title="CARRIAGE RETURN"><sub>CR</sub></code> (<span class="codepoint">U+000D</span>)
+        <li>Literals delimited by <code>'</code> (apostrophe, code point <span class="codepoint">U+0027</span>)
+          may not contain unescaped
+          <code>'</code> (apostrophe, code point <code class="codepoint">U+0027</code>),
+          <code>LF</code> (line feed, code point <code class="codepoint">U+000A</code>), nor
+          <code>CR</code> (carriage return, code point <code class="codepoint">U+000D</code>)
           characters.</li>
-        <li>Literals delimited by <code>"</code>, may not contain unescaped
-          <code>"</code>,
-          <code title="LINE FEED"><sub>LF</sub></code>,
-          nor <code title="CARRIAGE RETURN"><sub>CR</sub></code>
+        <li>Literals delimited by <code>&quot;</code>, may not contain unescaped
+          <code>&quot;</code> (quotation mark, code point <code class="codepoint">U+0022</code>),
+          <code>LF</code> (line feed, code point <code class="codepoint">U+000A</code>), nor
+          <code>CR</code> (carriage return, code point <code class="codepoint">U+000D</code>)
           characters.</li>
         <li>Literals delimited by <code>'''</code> may not contain the sequence of characters
           <code>'''</code>.</li>
```

Comment:
```suggestion
        <li>Literals delimited by a sequence of three <code>'</code> (apostrophe, code point <span class="codepoint">U+0027</span>) may not contain a sequence of three <code>'</code> (apostrophe, code point <span class="codepoint">U+0027</span>).</li>        
        <li>Literals delimited by a sequence of three <code>&quot;</code> (quotation mark, code point <code class="codepoint">U+0022</span>) may not contain a sequence of three <code>&quot;</code> (quotation mark, code point <code class="codepoint">U+0022</span>).</li>
```

Reply:
Although, maybe these should be rewritten with the `two concatenated <code>&lt;</code> characters, each having the code point <code class="codepoint">U+003C</code>` pattern?